cdap-ui/app/cdap/components/Header/ProductDropdown/ProductDropdown.scss

```diff
@@ -192,15 +192,15 @@ $cdap_mode_color: #cccccc;
     }
   }
 
-  .beta-badge {
-    font-size: 9px;
-    background-color: #5cb85c;
-    border-radius: 4px;
-    color: white;
-    margin-left: 2px;
-    padding: 2px 3px;
-    position: relative;
-    top: -7px;
-  }
+  // .beta-badge {
+  //   font-size: 9px;
+  //   background-color: #5cb85c;
+  //   border-radius: 4px;
+  //   color: white;
+  //   margin-left: 2px;
+  //   padding: 2px 3px;
+  //   position: relative;
+  //   top: -7px;
+  // }
```

Comment:
Will be removed or location moved depending on whether we want to add beta tag to the navbar or not.